TACC/Core-Portal
# bug/FP-1928 Fix Data Files Dropdown CSS

## Overview

Fixing the css for the data files dropdown that got removed due to a previous commit

## Related

* [FP-1928](https://jira.tacc.utexas.edu/browse/FP-1928)

## Changes

- Added the data-files-sidebar class to the Sidebar div
- Removed unused css in DataFilesSidebar.scss

## Testing

1. Go to Data Files in the portal 
2. Click on the Add button
3. Ensure css styling looks correct

## UI

Before: 

<img width="215" alt="Screen Shot 2022-11-22 at 10 06 30 AM" src="https://user-images.githubusercontent.com/23081932/203363379-931c3d71-a4aa-47d7-a72d-9fd8115000c4.png">

After: 

<img width="215" alt="Screen Shot 2022-11-22 at 10 06 46 AM" src="https://user-images.githubusercontent.com/23081932/203363411-3270cc8d-6b31-4086-b414-48b6ba35e7bc.png">
